.pre-commit-config.yaml

```diff
@@ -6,3 +6,15 @@ repos:
     -   id: end-of-file-fixer
     -   id: check-merge-conflict
     -   id: flake8
+
+-   repo: https://github.com/PyCQA/bandit
+    rev: '1.6.2'
+    hooks:
+    -   id: bandit
+
+-   repo: local
```

Comment:
Another suggested hook that runs the existing test suite before applying the commit, possibly catching a build failure in TravisCI

Reply:
Given how long the zipline test suite takes to run right now, I don't think I'd want to require people to run the test suite before every commit. I'd rather work toward speeding up the test suite and/or our CI first.